pkg/controllers/managementuser/networkpolicy/netpol.go

```diff
@@ -186,9 +186,12 @@ func (npmgr *netpolMgr) handleHostNetwork(clusterNamespace string) error {
 		return fmt.Errorf("couldn't list nodes err=%v", err)
 	}
 
-	cni, err := npmgr.getClusterCNI(clusterNamespace, nodes[0])
-	if err != nil {
-		return err
+	var cni string
+	if len(nodes) > 0 {
+		cni, err = npmgr.getClusterCNI(clusterNamespace, nodes[0])
+		if err != nil {
+			return err
+		}
```

Comment:
```suggestion
	var node *corev1.Node
	if len(nodes) > 0 {
		node = nodes[0]
	}

	cni, err = npmgr.getClusterCNI(clusterNamespace, node)
	if err != nil {
		return err
	}
```
Basically what @snasovich suggested, but as code :smile:

Reply:
Done.